sktime/forecasting/hf_transformers_forecaster.py

```diff
@@ -291,17 +342,50 @@ def _predict(self, fh, X=None):
             ),
         )
 
-        pred = pred.sequences.mean(dim=1).detach().cpu().numpy().T
+        if isinstance(self._y.index, pd.MultiIndex):
```

Comment:
Would it be possible to enforce that the index has to be always a pd.MultiIndex or at to cast it at the beginning of the _predict method? 

In this case no if-else case would be necessary and the code would become simpler.


@fkiraly are there casts that are applied automatically in the boilerplate layer?

Reply:
casted to pd.MultiIndex at the beginning of the _predict method